pom.xml

```diff
@@ -323,6 +324,12 @@
       <artifactId>reflections</artifactId>
       <version>0.10.2</version>
     </dependency>
+    <dependency>
+      <groupId>jakarta.annotation</groupId>
+      <artifactId>jakarta.annotation-api</artifactId>
+      <version>1.3.5</version>
+      <scope>provided</scope>
+    </dependency>
```

Comment:
what is a reason of this dependency ?

we are a library and any new dependency put in our dependencies is pain for our users.

Reply:
yes sorry need to revert.

Comes from `org.openrewrite.java.migrate.UpgradeToJava17`.

This is missing considering the recipe. IDK why it is, and its not needed kind of, but im sure they have a good reason to add it.